docs/UPSTREAMING.md

```diff
@@ -78,7 +76,7 @@ In the function `grub_install_remove_efi_entries_by_distributor`, sets the defau
 
 #### 0006-export-vars.patch
 
-Exports current grub setting vars. Need and usage is unclear.
+Exports current grub setting vars. This is required for our use in grub.cfg where we [set global variable from submenus](../pkg/grub/rootfs.cfg).
```

Comment:
Ah, we use submenus to set vars before booting. Interesting. I think we should try to get this upstreamed. If they push back, they may have a different way of doing the same thing.